sdk/identity/azure_identity/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 ### Breaking Changes
 
+- Replaced `azure_core::process::Executor` with `azure_identity::process::Executor`.
```

Comment:
It was never exposed so how is it a breaking change? It's just an implementation detail.

Reply:
The type of e.g. `AzureCliCredentialOptions.executor` has changed